debugging.md

```diff
@@ -0,0 +1,68 @@
+# Fehlerbehebung und Debugging
+
+Auch eine REDAXO-Installation kann einmal „Schluckauf“ haben. Im Folgenden werden Möglichkeiten aufgezeigt, deine REDAXO-Installation zu reparieren.
+
+> **Hinweis:** Wir empfehlen auch vor einem Fehler, regelmäßige Backups zu erstellen. Datenbank-Backups können automatisiert über das Cronjob-Addon erstellt werden.
+
+## Fehleranalyse: Whooops und Oooops
+
+![Ooops](/assets/v5.10.0-debug_ooops.png)
+
+Tritt ein Fehler auf, meldet REDAXO sich standardmäßig im Frontend mit einem Oooops und im Backend mit einem Rrrrroar.
+
+![Ooops](/assets/v5.10.0-debug_whooops.png)
+
+Wenn ein Administrator eingeloggt ist, oder der Administrator den  Debug-Modus aktiviert hat, wird anstelle des Oooops ein Whoooops mit genauerer Fehlerbeschreibung und Stacktrace ausgegeben.
+
+## Fehleranalyse: Die system.log-Datei
+
+![Screenshot /System/Logdateien](/assets/v5.10.0-debug_syslog.png)
+
+In der Datei `redaxo/data/core/system.log` werden Fehler geloggt - möglicherweise ist der Fehler bereits dabei.
+
+## Debugging: Einstellungen in der `config.yml`
+
+In der Datei `redaxo/data/core/config.yml` können Parameter zum Debugging aktiviert werden. Der Debug-Modus sollte ausschließlich nur dann aktiviert werden, wenn die Website nicht öffentlich erreichbar ist.
+
+## Debugging: Das debug-Addon
+
+![Debug-AddOn ab REDAXO 5.11](/assets/v5.11.0-debug_addon.png)
+
+Neu in REDAXO 5.11 ist das Debug-Addon, welches im Backend zusätzlich aktiviert werden kann. Es erscheint ein neuer Menüpunkt, in dem Clockwork gestartet wird. Jeder weitere Aufruf im Frontend oder Backend übergibt an Clockwork Debugging-Informationen.

```

Comment:
Neu in… vermeiden. Die aktuelle Entwicklung scheint auch darauf hinzuweisen, dass Gregor es auch für ältere Versionen verfügbar machen will.